### ISSUE TITLE
support for env_file

### ISSUE THREAD

**Issue by surajssd** [docker/libcompose#335]
When using env_file in docker-compose file, libcompose panics.

This is `docker-compose.yml` file I am using
```yaml
$ cat docker-compose.yml
version: "2"

services:
  mariadb:
    image: centos/mariadb
    ports:
      - "$DB_PORT"
    env_file: ./env1

  etherpad:
    image: centos/etherpad
    ports:
      - "80:9001"
    depends_on:
      - mariadb
    env_file: ./env2
    environment:
      DB_HOST: $DB_HOST
      DB_PORT: $DB_PORT
```

and environment files used in the above `docker-compose.yml` file

```bash
$ cat env1 
MYSQL_ROOT_PASSWORD=etherpad
MYSQL_DATABASE=etherpad
MYSQL_PASSWORD=etherpad
MYSQL_USER=etherpad

$ cat env2
DB_DBID=etherpad
DB_PASS=etherpad
DB_USER=etherpad

$ cat envs
DB_HOST=mariadb
DB_PORT=3306
```

running the application

```bash
$ export $(cat envs)
$ ./libcompose up
WARN[0000] Note: This is an experimental alternate implementation of the Compose CLI (https://github.com/docker/compose) 
panic: interface conversion: interface is string, not []interface {} [recovered]
	panic: interface conversion: interface is string, not []interface {}

goroutine 1 [running]:
panic(0x9f1b40, 0xc8202abcc0)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/docker/libcompose/vendor/github.com/urfave/cli.HandleAction.func1(0xc8201979f8)
	/home/vagrant/work/src/github.com/docker/libcompose/vendor/github.com/urfave/cli/app.go:472 +0x417
panic(0x9f1b40, 0xc8202abcc0)
	/usr/local/go/src/runtime/panic.go:443 +0x4e9
github.com/docker/libcompose/config.readEnvFile(0x7fa424bf2218, 0x1027268, 0xb23d50, 0x12, 0xc8202d4b40, 0x8, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/config/merge.go:89 +0x161
github.com/docker/libcompose/config.parseV2(0x7fa424bf2218, 0x1027268, 0x7fa424bf2298, 0xc8202c2f00, 0xb23d50, 0x12, 0xc8202d4b40, 0xc8202d4ae0, 0xfe1790, 0xc82000e880, ...)
	/home/vagrant/work/src/github.com/docker/libcompose/config/merge_v2.go:95 +0x83
github.com/docker/libcompose/config.MergeServicesV2(0xc8202c2f40, 0x7fa424bf2298, 0xc8202c2f00, 0x7fa424bf2218, 0x1027268, 0xb23d50, 0x12, 0xc8202a5500, 0x12b, 0x32b, ...)
	/home/vagrant/work/src/github.com/docker/libcompose/config/merge_v2.go:36 +0x2e3
github.com/docker/libcompose/config.Merge(0xc8202c2f40, 0x7fa424bf2298, 0xc8202c2f00, 0x7fa424bf2218, 0x1027268, 0xb23d50, 0x12, 0xc8202a5500, 0x12b, 0x32b, ...)
	/home/vagrant/work/src/github.com/docker/libcompose/config/merge.go:40 +0x24c
github.com/docker/libcompose/project.(*Project).load(0xc8202d2b60, 0xb23d50, 0x12, 0xc8202a5500, 0x12b, 0x32b, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/project/project.go:178 +0xe5
github.com/docker/libcompose/project.(*Project).Parse(0xc8202d2b60, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/project/project.go:86 +0x335
github.com/docker/libcompose/docker.NewProject(0xc8200aa620, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/docker/project.go:75 +0x7f1
github.com/docker/libcompose/cli/docker/app.(*ProjectFactory).Create(0x1027268, 0xc8202cc500, 0x0, 0x0, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/cli/docker/app/factory.go:33 +0x2d4
github.com/docker/libcompose/cli/app.WithProject.func1(0xc8202cc500, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/cli/app/app.go:44 +0x6e
reflect.Value.call(0x90fc40, 0xc8202c25c0, 0x13, 0xab4680, 0x4, 0xc820197958, 0x1, 0x1, 0x0, 0x0, ...)
	/usr/local/go/src/reflect/value.go:435 +0x120d
reflect.Value.Call(0x90fc40, 0xc8202c25c0, 0x13, 0xc820197958, 0x1, 0x1, 0x0, 0x0, 0x0)
	/usr/local/go/src/reflect/value.go:303 +0xb1
github.com/docker/libcompose/vendor/github.com/urfave/cli.HandleAction(0x90fc40, 0xc8202c25c0, 0xc8202cc500, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/vendor/github.com/urfave/cli/app.go:481 +0x2ee
github.com/docker/libcompose/vendor/github.com/urfave/cli.Command.Run(0xab9c20, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0xb1a9f0, 0x15, 0x0, ...)
	/home/vagrant/work/src/github.com/docker/libcompose/vendor/github.com/urfave/cli/command.go:186 +0x1301
github.com/docker/libcompose/vendor/github.com/urfave/cli.(*App).Run(0xc820001980, 0xc82000a260, 0x2, 0x2, 0x0, 0x0)
	/home/vagrant/work/src/github.com/docker/libcompose/vendor/github.com/urfave/cli/app.go:235 +0xa9c
main.main()
	/home/vagrant/work/src/github.com/docker/libcompose/cli/main/main.go:46 +0x19e4
```

libcompose version
```bash
$ ./libcompose version
WARN[0000] Note: This is an experimental alternate implementation of the Compose CLI (https://github.com/docker/compose) 
Version:      0.3.0-dev (HEAD)
Go version:   go1.6.2
Built:        
OS/Arch:      linux/amd64
DEPRECATED Action signature.  Must be `cli.ActionFunc`.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature
```

**Comment by joshwget** [comment]
This should be fixed by #307

**Comment by vdemeester** [comment]
It is fixed (tried it out, no panic 👼)
